### PR TITLE
docs: adjust manual github action screenshot in contributing.md

### LINKED ISSUE #15407: [bug] pixelated image on /docs
This image is huge and pixelated. If we decrease the image size the image quality should work fine and it will be easier to grasp the information for the reader. 
URL: https://coder.com/docs/CONTRIBUTING
Image

### PR DESCRIPTION
**Pull request by EdwardAngert**
resolves #15407

adjust manual github action screenshot in contributing.md

[preview](https://coder.com/docs/@15407-contrib-image/CONTRIBUTING#deploying-a-pr) (once ready)

### CODE REVIEW

**Review by matifali — APPROVED** [review]

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -98,10 +98,12 @@ Use the following `make` commands and scripts in development:
...
-1. By running `./scripts/deploy-pr.sh`
-2. By manually triggering the
-   [`pr-deploy.yaml`](https://github.com/coder/coder/actions/workflows/pr-deploy.yaml)
-   GitHub Action workflow ![Deploy PR manually](./images/deploy-pr-manually.png)
+- Run `./scripts/deploy-pr.sh`
+- Manually trigger the
+  [`pr-deploy.yaml`](https://github.com/coder/coder/actions/workflows/pr-deploy.yaml)
+  GitHub Action workflow:
+
+  <Image width="350px" src="./images/deploy-pr-manually.png" alt="Deploy PR manually" align="center" />
```
Comment (EdwardAngert) [inline docs/CONTRIBUTING.md:106]:
```suggestion
  <Image max-width:100% src="./images/deploy-pr-manually.png" alt="Deploy PR manually" align="center" />
```

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -103,7 +103,7 @@ this:
   [`pr-deploy.yaml`](https://github.com/coder/coder/actions/workflows/pr-deploy.yaml)
   GitHub Action workflow:
 
-  <Image src="./images/deploy-pr-manually.png" alt="Deploy PR manually" width="350px" align="center" />
+  <Image src="./images/deploy-pr-manually.png" alt="Deploy PR manually" height="348px" align="center" />
```
Comment (matifali) [inline docs/CONTRIBUTING.md:106]: 348. Nice.

Comment (EdwardAngert) [reply docs/CONTRIBUTING.md:106]: apparently `height` is the answer that doesn't get overridden 🤷

Comment (EdwardAngert) [reply docs/CONTRIBUTING.md:106]: shoutout to that HTML 4 book I had back in 2000

Comment (matifali) [reply docs/CONTRIBUTING.md:106]: loooks way better, Thank you

**Pull request merged**